modules/accounts/accounts.js

```diff
@@ -0,0 +1,224 @@
+const confFile = __dirname + '/../../config.json';
+const jsonfile = require('jsonfile');
+var eztz = {};
+var config = jsonfile.readFileSync(confFile);
```

Comment:
It can be a const

Reply:
these variables are not being constant throughout the code. Hence, defining as "var" is good.

Reply:
config is constant I think. I'm not asking for all of them, whichever are constant, make it const.